charts/consul/test/terraform/eks/main.tf

```diff
@@ -84,4 +87,86 @@ data "aws_eks_cluster" "cluster" {
 data "aws_eks_cluster_auth" "cluster" {
   count = var.cluster_count
   name  = module.eks[count.index].cluster_id
+}
+
+# The following resources are meant to apply when cluster_count=2 to set up vpc peering and the appropriate routes and
+# security groups so traffic between vpc's is allowed. Since the pipeline only ever deploys with a cluster_count of 2,
+# these resources are hardcoded to work in an environment with 2 clusters.
```

Comment:
🙏 thanks for the clarification. I was about to ask about it 😁

Reply:
@ishustava is this true for AKS as well? i noticed in your PR that is doesn't assume that the cluster_count is 2 and computes it.

Reply:
🤔 yeah I didn't assume that. I left a comment about it above.

Reply:
Yeah for AKS, the cluster_count is allowed to be 1 or 2. In the current state of this EKS PR, cluster_count is only allowed to be 2, unless you make some changes to the terraform (like commenting out the whole vpc block). Because these EKS resources are like unidirectional, so I needed to set them up such that ingress rule from cluster 0 --> 1 works, and vice versa, so I couldn't just rely on the count variable for these resources.

Reply:
Yeah for AKS, the cluster_count is allowed to be 1 or 2. In the current state of this EKS PR, cluster_count is only allowed to be 2, unless you make some changes to the terraform (like commenting out the whole vpc block). Because these EKS resources are like unidirectional, so I needed to set them up such that ingress rule from cluster 0 --> 1 works, and vice versa, so I couldn't just rely on the count variable for these resources.

Reply:
But it looks like @ishustava 's suggestion for conditionalizing the count may work! terraform apply is chugging along, so 🤞 that destroy works alright too